algorithms/sumRecur.js

```diff
@@ -0,0 +1,12 @@
+const RecurSum=(array,i=0,total=0)=>{
+  if(i==array.length){
+    console.log(total);
+  }
+  else{
+    total = total + array[i];
+    i = i+1;
+   return RecurSum(array, i,total);
```

Comment:
you can increment your i and total in your function parameter itself, so that code will look a bit small.